ballerina/constants.bal

```diff
@@ -71,3 +76,14 @@ const string CANONICAL_URI_GENERATION_ERROR_MSG = "Error occured while generatin
 const string CANONICAL_QUERY_STRING_GENERATION_ERROR_MSG = "Error occured while generating canonical query string.";
 const string XML_EXTRACTION_ERROR_MSG = "Error occurred while accessing the XML payload from the http response.";
 const string BINARY_CONTENT_EXTRACTION_ERROR_MSG = "Error occured while accessing binary content from the http response";
+const EXPIRATION_TIME_ERROR_MSG = "Invalid expiration time. Expiration time should be a positive integer.";
+const EMPTY_OBJECT_NAME_ERROR_MSG = "Invalid object name. Object name should not be empty.";
+const EMPTY_BUCKET_NAME_ERROR_MSG = "Invalid bucket name. Bucket name should not be empty.";
+
+# The action to be carried out on the object.
+# 'CREATE' - Create a new object
+# 'RETRIEVE' - Retrieve an existing object
+public enum ObjectAction {
+    CREATE,
+    RETRIEVE
+};
```

Comment:
```suggestion
# The action to be carried out on the object.
public enum ObjectAction {
    # Create a new object
    CREATE,
    # Retrieve an existing object
    RETRIEVE
};
```

This will properly document each constant.